### ISSUE TITLE
Processing time

### ISSUE THREAD

**Issue by songming1995** [lizimo061/PyLOAM#2]
Thanks for your contribution to the comminity. I wonder if this python-based LOAM can run in real-time? From my experience, it takes several minutes to process one frame.

**Comment by lizimo061** [comment]
Right, it cannot be run in real-time now. I try to optimize it for better performance but I guess I still need more time.

**Comment by YQChen8** [comment]
Thanks for your contribution to the comminity. I wonder if this python-based LOAM can run in ROS?

**Comment by lizimo061** [comment]
With some modifications I think yes? need to wrap the functions into ROS-compliant format.

**Comment by YQChen8** [comment]
Thanks，but I am a fresh learner now. Could you give me some advices on implementation of LOAM based Python and ROS?

**Comment by YQChen8** [comment]
Most of the original implementation of LOAM is written in C++, and the ROS implementation is also based on C++. I wonder if    there is a set of code that can implement the LOAM based ROS and Python.

**Comment by YQChen8** [comment]
Do you have WeChat, Can I ask you some questions? Thanks!
